gradle-diff-task-dependencies/plugin/src/main/groovy/com/novoda/gradle/diffdependencytask/DiffDependencyTaskPlugin.groovy

```diff
@@ -0,0 +1,28 @@
+package com.novoda.gradle.diffdependencytask
+
+import com.novoda.gradle.diffdependencytask.ext.ExtensionRegister
+import org.gradle.api.Plugin
+import org.gradle.api.Project
+
+class DiffDependencyTaskPlugin implements Plugin<Project> {
+
+    @Override
+    void apply(Project project) {
+
+        ExtensionRegister.INSTANCE.run()
```

Comment:
This is needed to make the `Task::ifDiffMatches` method available as soon as the plugin is applied.